### PR TITLE
TINY-10878: Fix issue with emojis not loading due to broken CDN link

### PR DESCRIPTION
**Pull request by ArvinJ-H**
Related Ticket: 
TINY-10878

Description of Changes:
- original CDN no longer valid, changed to a new one

Pre-checks:
* [x] Changelog entry added
* [x] <s>Tests have been added (if applicable)</s>
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] Docs ticket created (if applicable)

GitHub issues (if applicable):

### CODE REVIEW

File: modules/tinymce/src/plugins/emoticons/main/ts/api/Options.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ const register = (editor: Editor, pluginUrl: string): void => {
 
   registerOption('emoticons_images_url', {
     processor: 'string',
-    default: 'https://twemoji.maxcdn.com/v/13.0.1/72x72/'
+    default: 'https://cdn.jsdelivr.net/gh/twitter/twemoji@14.0.2/assets/72x72/'
```
Comment (TheSpyder) [inline modules/tinymce/src/plugins/emoticons/main/ts/api/Options.ts:43]: I think I'd trust cloudflare over jsdelivr at this point.
```suggestion
    default: 'https://cdnjs.cloudflare.com/ajax/libs/twemoji/15.1.0/72x72/'
```

Cloudflare have also automatically redirected the `twemoji` link to the [new maintenance home](https://github.com/jdecked/twemoji) after twitter fired the developers working on it.

**Review by TheSpyder — APPROVED** [review]

**Review by ltrouton — APPROVED** [review]

File: modules/tinymce/src/plugins/emoticons/main/ts/api/Options.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ const register = (editor: Editor, pluginUrl: string): void => {
 
   registerOption('emoticons_images_url', {
     processor: 'string',
-    default: 'https://twemoji.maxcdn.com/v/13.0.1/72x72/'
+    default: 'https://cdnjs.cloudflare.com/ajax/libs/twemoji/15.1.0/72x72/'
```
Comment (spocke) [inline modules/tinymce/src/plugins/emoticons/main/ts/api/Options.ts:43]: I'm surprised that our product works like this. I mean this isn't really on-prem if you enable this it goes out on the internet for things.

**Review by spocke — APPROVED** [review]

File: modules/tinymce/src/plugins/emoticons/main/ts/api/Options.ts (hunk 2 above)
Comment (TheSpyder) [reply modules/tinymce/src/plugins/emoticons/main/ts/api/Options.ts:43]: It is documented, though.
https://www.tiny.cloud/docs/tinymce/latest/emoticons/#emoticons_images_url

Speaking of which, we should change the docs URL for downloading the image assets to the new maintenance home as well... @ArvinJ-H can you please log a JIRA ticket for that

Comment (spocke) [reply modules/tinymce/src/plugins/emoticons/main/ts/api/Options.ts:43]: Right I think this is the only thing that works like this that depends on a third party url. It's just a bit odd. I see why we do it we don't want to download that huge thing and ship that. But I guess most people just use the unicode emojis.

Comment (ShiridiGandham) [reply modules/tinymce/src/plugins/emoticons/main/ts/api/Options.ts:43]: > Speaking of which, we should change the docs URL for downloading the image assets to the new maintenance home as well... @ArvinJ-H can you please log a JIRA ticket for that

@ArvinJ-H Could you please create a documentation ticket if it hasn't been done yet?

Comment (ArvinJ-H) [reply modules/tinymce/src/plugins/emoticons/main/ts/api/Options.ts:43]: yes, there was a ticket logged to update the url in doc, forgot to link it in ticket.

https://ephocks.atlassian.net/browse/DOC-2406

**Pull request closed without merging**